styles.css

```diff
@@ -451,3 +443,286 @@ form {
   color: white;
   background-color: #7f8cff;
 }
+
+@media all and (max-width: 768px) {
+  #header .nav_icon ul {
+    display: none;
+  }
+
+  .for-big-size {
+    display: none;
+  }
+}
+
+/* desktop version */
+
+@media all and (min-width: 768px) {
+  #header {
+    padding: 0;
+    margin: 25px 0 0 155px;
+  }
+
+  /* page one */
+
+  .frame-2 ul {
+    margin-left: 0;
+  }
+
+  .tags ul {
+    margin-left: 0;
+  }
+
+  .language-list ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .language-list ul li {
+    flex-direction: column;
+    max-width: 122px;
+    max-height: 120px;
+    padding: 2% 7% 25% 3%;
+  }
+
+  #third-page ul {
+    margin: 0 auto 32px 48px;
+  }
+
+  #header .nav_icon ul {
+    display: flex;
+    list-style: none;
+    margin-right: 154px;
+  }
+
+  #header .nav_icon ul li {
+    margin-right: 32px;
+  }
+
+  #header .nav_icon ul li:last-of-type {
+    margin-right: 0;
+  }
+
+  #header .nav_icon a {
+    display: none;
+  }
+
+  #first_page {
+    background-image: url(images/Header\ bg@2x.png);
+    background-size: cover;
+  }
+
+  .cover {
+    background-image: none;
+  }
+
+  .title h1 {
+    margin-left: 0;
+    padding: 0;
+  }
+
+  #first_page h1 {
+    margin-left: 260px;
+    padding-top: 281px;
+    margin-bottom: 12px;
+  }
+
+  .upper-part p {
+    margin-left: 0;
+    margin-top: 12px;
+    max-width: 447px;
+  }
+
+  .contact p {
+    margin-left: 436px;
+    margin-right: 434px;
+    margin-top: 12px;
+    margin-bottom: 12px;
+  }
+
+  #first_page p {
+    margin-left: 260px;
+    margin-bottom: 12px;
+  }
+
+  #first_page .connect {
+    margin-left: 260px;
```

Comment:
- [ ] I think it will be better if you write your 'LET'S CONNECT' text in uppercase as shown in the Figma design. This will make your UI look better 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/194622389-f3e8e201-d1d3-4071-ba90-51b26bd594d1.png)  | ![image](https://user-images.githubusercontent.com/63473584/194622310-2c715f3c-8e68-438b-8306-301d852efd75.png)  |